packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -412,16 +416,46 @@ const _BaseButton: React.ForwardRefRenderFunction<BladeElementRef, BaseButtonPro
     buttonTypographyTokens: buttonTypography,
     children: childrenString,
     isDisabled: disabled,
-    size,
-    variant,
+    size: buttonGroupProps.size ?? size,
+    variant: buttonGroupProps.variant ?? variant,
     theme,
-    color,
+    color: buttonGroupProps.color ?? color,
```

Comment:
expected to give ButtonGroup a priority over individual Button?

Reply:
Yes, all buttons should have the same color in a button group.